src/blocks/team/WithOverlappedCards/WithOverlappedCards.js

```diff
@@ -46,14 +49,14 @@ const WithOverlappedCards = () => {
             align={'center'}
             fontWeight={700}
           >
-            Our team
+            Case Studies
           </Typography>
           <Typography fontWeight={700} variant={'h4'} align={'center'}>
             Trust the professionals
```

Comment:
This should come from a default value on the component, like title='Trust the Professionals', then it has the option to be come editable in different pages and sections it loads. (I see this component being used a lot)